scripts/package

```diff
@@ -13,16 +13,63 @@ fi
 
 cp -r bin package/
 
-APIVERSION=`./bin/longhorn-share-manager version --client-only|jq ".clientVersion.apiVersion"`
+APIVERSION=""
```

Comment:
What is the purpose of initializing the variable here?

Reply:
It is redundant. Removed. Thank you.